doc/org/build/networking.md

```diff
@@ -0,0 +1,256 @@
+
+-   A `prefabName`, to indicate which prefab to initialize,
+-   An `initialData` struct, to know how to initialize the prefab.
+
+Prefabs must be registered in `prefabs` , a map from `PrefabName` to `PrefabDefinition`.
+
+`PrefabDefinition` s include `template` functions that take `InitialData` and return `EntityDef` s. `EntityDef` s are defined in `jsx`, using the `createElementEntity` transformer (not `React` &rsquo;s `createEntity` transformer).
+
+For example, the commonly used `"media"` prefab&rsquo;s `template` is:
+
+    export function MediaPrefab(params: MediaLoaderParams): EntityDef {
+      return (
+        <entity
+          name="Interactable Media"
+          networked
+          networkedTransform
+          mediaLoader={params}
+          deletable
+          grabbable={{ cursor: true, hand: true }}
+          destroyAtExtremeDistance
+          floatyObject={{
+            flags: FLOATY_OBJECT_FLAGS.MODIFY_GRAVITY_ON_RELEASE,
+            releaseGravity: 0
+          }}
+          networkedFloatyObject={{
+            flags: FLOATY_OBJECT_FLAGS.MODIFY_GRAVITY_ON_RELEASE
+          }}
+          rigidbody={{ collisionGroup: COLLISION_LAYERS.INTERACTABLES, collisionMask: COLLISION_LAYERS.HANDS }}
+          physicsShape={{ halfExtents: [0.22, 0.14, 0.1] }} /* TODO Physics shapes*/
+          scale={[1, 1, 1]}
+        />
+      );
+    }
+
+When supplied with `MediaLoaderParams` as its `InitialData`, this prefab `template` creates an entity with several components, including a `Networked` and `NetworkedTransform` component.
+
+Calling `createNetworkedEntity(world, "media", mediaLoaderParams)` would cause the `networkSendSystem` to send a `CreateMessage` the next time it sends messages.
+
+
+<a id="org7110b85"></a>
+
+# Writing Networked Components
+
+Networked components are `Component` s that have been associated with a `NetworkSchema` in the `schemas` map.
+
+A `NetworkSchema` indicates how to pack networked component data into `UpdateMessage` s, and has the following properties:
+
+-   A `componentName` that uniquely identifies the component.
+-   A `serialize` (and `deserialize`) function that defines how component data is packed into (and unpacked from) `UpdateMessage` s.
+-   An optional `serializeForStorage` (and `deserializeForStorage`) function that defines how component data is packed into (and unpacked from) `StorableUpdateMessages` s, able to be saved (and loaded) from the database.
+
+The `serialize` and `deserialize` functions can be generated by calling `defineNetworkSchema`.
+
+The `serializeForStorage` and `deserializeForStorage` functions need careful authoring to allow for reading component state that has been saved to the database in a backwards-compatible way. More information about this will be written later.
+
+Note that `NetworkSchema` s are likely to change in the near future, as we are looking for ways to simplify the complexity that `serializeForStorage` and `deserializeForStorage` introduce.
```

Comment:
I have not reviewed bitECS to know if this is true, but the community may expect that they need to still do something with hub.html to write new networked components. It probably isn't appropriate here, but if hub.html is still being used we should let people know that the templates are no longer required